tasks/systray.py

```diff
@@ -82,4 +89,7 @@ def clean(ctx):
 
     # remove the bin/agent folder
     print("Remove systray executable")
-    ctx.run("rm -rf ./bin/agent/ddtray.exe")
+    try:
+        os.remove('./bin/agent/ddtray.exe')
+    except Exception as e:
+        print(e)
```

Comment:
Is it a debug code?

Reply:
I was just replacing the `rm -rf` shell command